src/main/webapp/onClickEvents.js

```diff
@@ -0,0 +1,42 @@
+/**Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+// add choosen filter to search bar and fetch search results from database
+addFilterToSearch = (searchString) => {
+    const searchBar = document.getElementById(SEARCH_BAR);
+    searchBar.value += " " + searchString;
```

Comment:
Why are we adding " " at the start?

Reply:
if there is something already in search bar we just need to append filter to that so a space is needed to separate two words otherwise full text search will treat them as one